File: app/src/main/java/com/deviprasaddayal/depiciat/listeners/OnFileActionListener.java
package com.deviprasaddayal.depiciat.listeners;

import java.io.File;

public interface OnFileActionListener {
    void onFilePathCreatedForCamera(String imagePath, File imageFile);

    void onFileAddRequest();

    void onFileViewRequest(int position);

    void onFileDeleted(int position, String removedFileName);
}

File: app/src/main/java/com/deviprasaddayal/depiciat/listeners/OnFileActionListener.kt
package com.deviprasaddayal.depiciat.listeners

import java.io.File

interface OnFileActionListener {
    fun onFilePathCreatedForCamera(imagePath: String, imageFile: File)

    fun onFileAddRequest()

    fun onFileViewRequest(position: Int)

    fun onFileDeleted(position: Int, removedFileName: String)
}
